handbook/product/product_management/direction_template.md

```diff
@@ -0,0 +1,75 @@
+# <Product> direction
+
+This page outlines the vision, strategy and goals of the <Product> team.
+
+# Vision
+
+#### 1 Year vision
+
+> Some vision
+
+#### 3 Year vision
+
+> Some vision
+
+#### 10 year vision
+
+> Some vision
+
+## Mission
+
+TODO
+
+# Where we are now
```

Comment:
I would propose removing this as a separate heading, and making it part of the introduction for each direction page.